src/bgw/job_stat.c

```diff
@@ -207,7 +207,7 @@ calculate_next_start_on_failure(TimestampTz finish_time, int consecutive_failure
 	MemoryContext oldctx;
 	if (!IS_VALID_TIMESTAMP(finish_time))
 	{
-		elog(LOG, "calculate_next_start_on_failure, got bad finish_time");
+		elog(LOG, "%s: invalid finish time", __func__);
```

Comment:
it is slightly unsync with the message below, Is that a debug message? Should this be an error?